graspy/simulations/__init__.py

```diff
@@ -1,3 +1,16 @@
-from .simulations import *
-from .simulations_corr import *
+from .simulations import sample_edges, er_np, er_nm, sbm, rdpg, p_from_latent
+from .simulations_corr import sample_edges_corr, er_corr, sbm_corr
 from .rdpg_corr import rdpg_corr
+
+__all__ = [
+    "sample_edges",
+    "er_np",
+    "er_nm",
+    "sbm",
+    "rdpg",
+    "p_from_latent",
+    "sample_edges_corr",
+    "er_corr",
+    "sbm_corr",
+    "rdpg_corr",
+]
```

Comment:
Take a look at the travis build failure. There are some formatting errors;  try to run black on `__init__.py` and it should be resolved.

Reply:
That's strange, I'm pretty sure I already ran black on the file before PRing. I followed Pedigo's instructions for how to set up black in https://github.com/bdpedigo/giskard/blob/master/python_start.md

Reply:
Black has updated recently. Check which version you are using and/or try updating

Reply:
It says I'm running 19.10b0 in my conda environment

Reply:
install black>=20

Reply:
% conda update black

Collecting package metadata (current_repodata.json): done
Solving environment: done

All requested packages already installed.

% black --version
black, version 19.10b0

Not sure why I can't install black>=20, I also tried uninstalling and reinstalling black, but the same thing happened.

Reply:
try `pip install black==20.8b1`

Reply:
or `conda install -c conda-forge black`

Reply:
I did that, now my version of black is 20.8b1 when I'm not in my conda environment, but it's still 19.10b0 when I am

Reply:
try the conda version then? also, it gets a bit weird, but you can actually use pip from inside a conda environment

Reply:
oh wait nvm i didnt see the second command you said now my black version in my conda environment is 20.8b1

Reply:
I don't think the problem is coming from the formatting, it looks like the symmetrize function from utils can't be found, but I am not sure why this is because I followed the same format as the __init__.py file for embed, and the files in embed also imported some of the functions in utils as well.

Reply:
i am not sure how to help debug here, what have you tried?

Reply:
or maybe @asaadeldin11 can also try to help

Reply:
in case you didn't fully understand the issue (I tracked down the traceback through the VSCode debugger, which I highly recommend learning) --

1. `tests/test_sims.py` had `from graspy.simulations import *`
2. graspy.simulations had a `from .simulations import *`
3. graspy.simulations.simulations had `from ..utils import symmetrize, cartprod`
4. because everything was run with a `from x import *` command, neither of those functions were attached to sub-namespaces (e.g., `symmetrize` wasn't called `graspy.utils.symmetrize`, it was just `symmetrize`)

Reply:
see https://docs.python.org/3/reference/import.html for an explanation of python's pretty annoying import system